extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/process/ProcessConfigCreator.scala

```diff
@@ -32,4 +34,12 @@ trait ProcessConfigCreator extends Serializable {
   def classExtractionSettings(modelDependencies: ProcessObjectDependencies): ClassExtractionSettings =
     ClassExtractionSettings.Default
 
+  def modelSettings(modelDependencies: ProcessObjectDependencies): ModelSettings = {
```

Comment:
Let's do this differently, let's replace `ProcessObjectDependencies.config` with:
`val modelConfig: ModelConfig`, let's keep `rawConfig: Config` field inside `ModelConfig` with TODO that w should parse config to normal, class fields.